.github/workflows/will-it-blend-develop.yml

```diff
@@ -22,25 +22,7 @@ jobs:
         pg_version: ["pg11", "pg12", "pg13", "pg14", "pg15"]
         container: ["fedora:36", "debian:bullseye", "alpine:3.18", "amazon:2"]
     steps:
-      # If this workflow is being called from a schedule/cron job, then let's
-      # force the "develop" branch. Otherwise, use whatever is passed in via
-      # GITHUB_HEAD_REF. The result of this will be used below in the
-      # actions/checkout@vX step. Note that at the time of this writing, Github
-      # Actions does not allow us to specify which branch to run a schedule from
-      # (it always runs from the default branch, which in this case is master).
-      - name: Set up correct branch environment variable
-        run: |
-          if [ $GITHUB_EVENT_NAME == "schedule" ]; then
-            echo "Running via schedule, so using branch develop"
-            echo "NIGHTLY_BUILD_REF=develop" >> $GITHUB_ENV
-          else
-            echo "Not running via schedule, so using branch $GITHUB_HEAD_REF"
-            echo "NIGHTLY_BUILD_REF=$GITHUB_HEAD_REF" >> $GITHUB_ENV
-          fi
-
       - uses: actions/checkout@v3
-        with:
-          ref: ${{ env.NIGHTLY_BUILD_REF }}
```

Comment:
Sorry, somehow I overlooked my @ notification. My apologies, @thomcc 

I think removing this will be OK and should work once develop is set as the default branch. If things fall apart of course I can go in and correct some things. We will also need to be sure that the manual run of WIB will work after these changes go through.

Reply:
It already is, these are just simplifications of CI since we now don't need manual overrides and they... _should_ be no-ops?

Reply:
Yeah, it's best to remove the unnecessary things as to avoid confusion. I hated that I originally had to add in the logic above to make it work, and am delighted that it now gets to be ripped out.